ILRepack/Steps/ResourceProcessing/BamlResourcePatcher.cs

```diff
@@ -84,8 +84,10 @@ private void ProcessRecord(PropertyWithConverterRecord record, AssemblyDefinitio
 
         private void ProcessRecord(AssemblyInfoRecord record)
         {
+            var assemblyName = new System.Reflection.AssemblyName(record.AssemblyFullName);
+
             var assemblyDefinition = _otherAssemblies.FirstOrDefault(
-                asm => asm.Name.Name == record.AssemblyFullName || asm.Name.FullName == record.AssemblyFullName);
+                asm => asm.Name.Name == assemblyName.Name || asm.Name.FullName == record.AssemblyFullName);
 
             if (assemblyDefinition != null)
             {
```

Comment:
maybe the FullName should be used here, but I guess the simple name works as well ?